dashboard/test/mailers/previews/pd_teacher_application_preview.rb

```diff
@@ -0,0 +1,41 @@
+# This can be viewed on non-production environments at /rails/mailers/pd/teacher_application_mailer
+class Pd::TeacherApplicationMailerPreview < ActionMailer::Preview
+  include FactoryGirl::Syntax::Methods
```

Comment:
FactoryGirl isn't being used here (`create` or `build` to generate fixtures), so you can remove this line.